src/main/java/eu/yals/telegram/TelegramArguments.java

```diff
@@ -62,15 +65,15 @@ static Builder builder() {
         Builder() {
         }
 
-        Builder(String url) {
+        Builder(final String url) {
             this.urlString = url;
         }
 
         TelegramArguments buildEmpty() {
             return EMPTY_ARGS;
         }
 
-        Builder andDescription(String description) {
+        Builder andDescription(final String description) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/yadevee/yals/pullRequest?prid=5409395)